developer-docs-site/docs/sdks/sdk-overview.md

```diff
@@ -0,0 +1,129 @@
+
+The core SDK layer exposes the functionalities needed by most applications, including:
+
+- The key generation
+- Transaction signing and submission (through BCS and transaction builder)
+- Transaction status querying, and
+- Various kinds of information retrieval.
+
+The high-level APIs layer leverages the core SDK to provide convenient methods around **NFT tokens.**
+
+### **OpenAPI client**
+
+The OpenAPI client is a set of classes that are generated based on the Aptos REST API spec. See the OpenAPI YAML spec definition [here](https://github.com/aptos-labs/aptos-core/blob/main/api/doc/openapi.yaml).
+
+### **Aptos Account**
+
+Provides the methods for generating Ed25519 key pairs, signing a bytes buffer with an Ed25519 public key, and deriving initial account addresses from the public keys.
+
+### **BCS Library**
+
+A subset of BCS standards implemented in Typescript.
+
+### **Transaction builder**
+
+This contains the Typescript types for constructing transaction payloads. The Typescript SDK supports three kinds of transaction payloads:
+
+- ScriptFunction
+- Script
+- ModuleBundle
+
+### **Aptos Client**
+
+The main component of Typescript SDK. It exposes the methods for retrieving account resources, transactions, and modules.
+
+The Aptos Client component also supports two transaction signing and submission methods.
+
+- Submitting transactions in JSON format, which delegates the signing message (input to the signing function) creation to the
+  API server.
+- Submitting transactions in BCS format, which prepares and signs the raw transactions on the client-side. This method leverages the BCS Library and Transaction Builder for constructing the transaction payloads.
+
+:::notice
+The second method is the recommended way for submitting transactions to the Aptos chain.
+:::
+
+### **Token Client**
+
+Provides methods for NFT collection creation, token minting, and token querying.
+
+## Validation for Transaction Builder and BCS
+
+Signing and submitting BCS transactions is the core functionality of Aptos SDKs. The Transaction Builder and the BCS are used to assemble and serialize the transaction payloads for signing and submission.
+
+Given that different programming languages have different primitive type constraints (e.g., byte length, value range, etc.) and various composite types support (e.g., enum, struct, class, etc.), the code for data serialization is hard to validate.
+
+The Aptos SDKs provide two levels of validation for Transaction Builder and BCS.
+
+1. First, with the unit tests and end-to-end (e2e) tests.
+
+:::notice
+An example of unit tests for the BCS serializer can be found [here](https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/transaction_builder/bcs/serializer.test.ts)
+
+An example of an e2e test for submitting a BCS transaction can be found [here](https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/aptos_client.test.ts#L88)
+:::
+
```

Comment:
On links, like 107 and 109 and everywhere else, always put a period either after "here" like `[here.]` or after the closing parenthesis of the url, like `.../bcs/serializer.test.ts).`